metatron-app/metatron-discovery
# Internationalize mailing template

### Description

Internationalize mailing template
- Approved user registration
- Rejected user registration
- Change password

**Related Issue** : 

#1933 

### How Has This Been Tested?

1. Create a new account in the login page with a valid e-mail
2. Check your mailbox and confirm the messages are translated properly

### Types of changes
